client/src/main/scala/com/lunatech/undoredo/recipes/Client.scala

```diff
@@ -0,0 +1,57 @@
+package com.lunatech.undoredo.recipes
+
+import diode._
+import org.scalajs.dom.ext.Ajax
+import io.circe.parser._
+import io.circe.syntax._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.util.{ Failure, Success }
+
+import com.lunatech.undoredo.recipes.models._
+
+object Client {
+
+  val baseUrl = "http://localhost:5000"
```

Comment:
maybe this could be passed from backend via an attribute data-xxx in the index.html

Reply:
This will go away since the api will be in the akka-http server